src/choropleth.ts

```diff
@@ -78,16 +81,19 @@ let mapTimeoutRef;
 //
 export const mapBounds = {
   width: 975,
-  height: 610,
+  height: 710,
```

Comment:
Do these maps bounds work for all countries?  Or should we set them on a per country basis?

Reply:
Good catch! Changed it back. I originally wanted to increase the height of the map only for the UK, but now with the working map I can see that there is not much point in doing so. Forgot to change it back.